.evergreen/config.yml

```diff
@@ -695,60 +670,42 @@ functions:
         working_dir: "src"
         shell: "bash"
         script: |
-          ${PREPARE_SHELL}
           if [ "${skip_EC2_auth_test}" = "true" ]; then
              echo "This platform does not support the web identity auth test, skipping..."
              exit 0
           fi
-          set -ex
+
+          # Try to source exported AWS Secrets
```

Comment:
We can probably move this logic into `run-mongodb-aws-test.sh` and pass in the test to run (`web-identity` in this case).